drivers/pinctrl/pinctrl_bflb.c

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2021-2024 Gerson Fernando Budke <nandojve@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/drivers/pinctrl.h>
+#include <bflb_pinctrl.h>
+#include <bflb_glb.h>
+#include <bflb_gpio.h>
+
+/* clang-format off */
```

Comment:
can't we leave the clang-format on?

Reply:
clang mess a lot! I prefer keep it disabled.

Reply:
It is also possible to ignore the clang-format warnings on CI, as well as turn it off on your editor settings.
Hopefully this would improve over time.

This would allow to keep it "Linux coding-style friendly" :)
https://www.kernel.org/doc/html/v6.12-rc3/process/coding-style.html#editor-modelines-and-other-cruft